keycloak/keycloak-admin-ui
# Complete partial import functionality.

## Motivation
Completed Partial Import

## Brief Description
The import button is now active and it calls the REST endpoint.  Then it displays the results as per:
https://marvelapp.com/prototype/fg26f9b/screen/75624798

## Verification Steps
1. Go to `Realm settings`.
2. Select Partial import form the Action dropdown.
3. Import a previously exported Keycloak json. (Samples test data at `/cypress/fixtures/partial-import-test-data/`. kcexport.json is a good one to look at)
4. Verify that modal presents the user with options as shown in the design, plus multi-realm functionality.
5. Press import and see results.

## Checklist:

- [ ] Help has been implemented (not applicable)
